src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ArtifactsSizeTest.cs

```diff
@@ -0,0 +1,100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Formats.Tar;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests;
+
+[Trait("Category", "SdkContent")]
+public class ArtifactsSizeTest : SmokeTests
+{
+    private static readonly string BaselineFilePath = BaselineHelper.GetBaselineFilePath($"ArtifactsSizes/{Config.TargetRid}.txt");
+    private static readonly Dictionary<string, long> BaselineFileContent = new Dictionary<string, long>();
+    private static readonly Regex BuildVersionPattern = new(@"\b\d+\.\d+\.\d+[-@](alpha|preview|rc|rtm)\.\d(\.\d+\.\d+)?\b");
+    private const int SizeThresholdPercentage = 25;
+
+
+    public ArtifactsSizeTest(ITestOutputHelper outputHelper) : base(outputHelper)
+    {
+        if (File.Exists(BaselineFilePath))
+        {
+            string[] baselineFileContent = File.ReadAllLines(BaselineFilePath);
+            foreach (string entry in baselineFileContent)
+            {
+                string[] splitEntry = entry.Split(':');
+                BaselineFileContent[splitEntry[0].Trim()] = long.Parse(splitEntry[1].Trim());
+            }
+        }
+        else
+        {
+            Assert.True(Directory.Exists(BaselineHelper.GetBaselineFilePath("ArtifactsSizes/")));
```

Comment:
What value does this check add?  From a UX perspective, when the test fails, all you know is that an assert fails without a message indicating what was checked.  It seems like the following assert is sufficient in that it describes the baseline file doesn't exist which may include the containing folder.